planter/cloudbuild.yaml

```diff
@@ -11,7 +11,7 @@ steps:
             '.' ]
     dir: planter/
 substitutions:
-  _BAZEL_VERSION: '0.20'
+  _BAZEL_VERSION: '0.25.2'
```

Comment:
@Katharine WDYT, should this be a matrix?

also @ixdy we should probably update OWNERS for this.

Reply:
It already is; this PR won't do anything because this value is always overridden (see variants.yaml).

Reply:
@Katharine what do I need to do?

Reply:
Update `BAZEL_VERSION` in [`variants.yaml`](https://github.com/kubernetes/test-infra/blob/master/planter/variants.yaml) (and possibly also update `planter.sh` to match?)

Reply:
We do not want to bump the version in planter.sh till it is tested, but I will bump variants.